tests/Fixtures/inc/Engine/Preload/Activation/refreshOnUpdate.php

```diff
@@ -2,14 +2,23 @@
 return [
 	'versionInferiorShouldDoNothing' => [
```

Comment:
@engahmeds3ed you should update names from fixtures to keep them coherent.
```suggestion
	'versionSuperiorShouldReload' => [
```